code/test/util.js

```diff
@@ -215,6 +269,123 @@ const getEOTP = async ({ seed, hseed, effectiveTime, timeOffset }) => {
   return { index, eotp }
 }
 
+// transactionExecute commits and reveals a wallet transaction
+const transactionExecute = async ({ wallet, operationType, tokenType, contractAddress, tokenId, dest, amount, data, address, randomSeed, backlinkAddresses, testTime }) => {
+  if (testTime === undefined) { testTime = Date.now() }
+  // // calculate counter from testTime
+  const counter = Math.floor(testTime / INTERVAL)
+  const otp = ONEUtil.genOTP({ seed: wallet.seed, counter })
+  // // calculate wallets effectiveTime (creation time) from t0
+  const info = await wallet.wallet.getInfo()
+  const t0 = new BN(info[3]).toNumber()
+  const walletEffectiveTime = t0 * INTERVAL
+  const index = ONEUtil.timeToIndex({ effectiveTime: walletEffectiveTime, time: testTime })
+  const eotp = await ONE.computeEOTP({ otp, hseed: wallet.hseed })
+
+  // Format commit and revealParams based on tokenType
+  let commitParams
+  let revealParams
+  let paramsHash
+  console.log(`operationType: ${operationType}`)
```

Comment:
Logger?

Reply:
Yes, updated 🙏 
Also by naming the tests more effectively as proposed this will give us information around what is running e.g.
```
it('OPERATION 0 TRACK: must be able to track tokens', async () => {
  })
  ```